Source/Events.Handling/Builder/TypedEventHandlerMethod.cs

```diff
@@ -14,13 +14,25 @@ namespace Dolittle.SDK.Events.Handling.Builder
     public class TypedEventHandlerMethod<T> : IEventHandlerMethod
         where T : class
     {
-        readonly TypedEventHandlerSignature<T> _method;
+        readonly Func<T, EventContext, Task> _method;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TypedEventHandlerMethod{T}"/> class.
         /// </summary>
-        /// <param name="method">The <see cref="EventHandlerSignature" />.</param>
-        public TypedEventHandlerMethod(TypedEventHandlerSignature<T> method) => _method = method;
+        /// <param name="method">The <see cref="TypedEventHandlerSignature{T}" />.</param>
+        public TypedEventHandlerMethod(TypedEventHandlerSignature<T> method)
+            => _method = (T @event, EventContext context) => method(@event, context);
```

Comment:
Doesn't this method already match the signature?

Reply:
This should be resolved by latest commits